src/components/BannerSlider/index.tsx

```diff
@@ -1,85 +1,70 @@
 import { useEffect, useState } from 'react';
 import {
+  Banner,
   LeftArrow,
   RightArrow,
   useInterval,
   useTypedTranslation,
-} from '@src/shared';
+} from '@shared';
 import classNames from 'classnames';
-
-interface BannerMockType {
-  img: string;
-  link: string;
-  title: string;
-  description: string;
-}
-
-const bannerMock: BannerMockType[] = [
-  {
-    img: 'pink',
-    link: 'catalog/pink',
-    title: 'Sales',
-    description: 'Go here and buy it PINK!',
-  },
-  {
-    img: 'green',
-    link: 'catalog/green',
-    title: 'Sales',
-    description: 'Go here and buy it GREEN!',
-  },
-  {
-    img: 'blue',
-    link: 'catalog/blue',
-    title: 'Sales',
-    description: 'Go here and buy it BLUE!',
-  },
-];
+import axios from 'axios';
 
 export default function BannerSlider(): JSX.Element {
   const [bannerIndex, setBannerIndex] = useState(0);
-  const [banner, setBanner] = useState(bannerMock[0]);
+  const [banners, setBanners] = useState<[] | Banner[]>([]);
+  const [banner, setBanner] = useState(banners[0]);
   const t = useTypedTranslation();
 
   const setDirection = (direction: 'back' | 'forward'): number => {
     switch (direction) {
       case 'back':
-        return bannerIndex > 0 ? bannerIndex - 1 : bannerMock.length - 1;
+        return bannerIndex > 0 ? bannerIndex - 1 : banners.length - 1;
       case 'forward':
-        return bannerIndex < bannerMock.length - 1 ? bannerIndex + 1 : 0;
+        return bannerIndex < banners.length - 1 ? bannerIndex + 1 : 0;
     }
   };
 
   useEffect(() => {
-    setBanner(bannerMock[bannerIndex]);
+    setBanner(banners[bannerIndex]);
   }, [bannerIndex]);
 
-  useInterval(() => setBannerIndex(setDirection('forward')), 5000);
+  useEffect(() => {
+    async function getBanners() {
+      const { data } = await axios.get('https://alex-online-store.fly.dev/api/v1/shop/guest_user/banner_list/');
```

Comment:
Please, move this api string to .env file and put it in VITE_APP_API_URL key. Then we will use it in src/app/api/base.ts
Also create new api point inside src/app/api/ folder which will get information from backend about banners. See example src/app/api/auth.ts